index.html

```diff
@@ -86,80 +91,47 @@
+
       <div class="note">
-        The `"next inert"` <a>loadpolicy</a> is equivalent to `rel=prefetch` implemented by some browsers; `"next"` <a>loadpolicy</a> is semantically equivalent to `rel=prerender` implemented by some browsers. This specification standardizes and extends previous prefetch and prerender functionality with additional capabilities.
+        To ensure compatibility and improve the success rate of prerendering requests the target page can use the [[PAGE-VISIBILITY]] to determine the visibility state of the page as it is being rendered and implement appropriate logic to avoid actions that may cause the prerender to be abandoned (e.g. non-idempotent requests), or unwanted side-effects from being triggered (e.g. analytics beacons firing prior to the page being displayed).
       </div>
-
-      <section>
-        <h2>`next` load and processing policy</h2>
-
-        <p>This specification extends the <code><dfn>loadpolicy</dfn></code> of <code><a>preload</a></code> relation, which consists of a space-separated set of keywords. These keywords determine the load and processing policy of the specified resource:</p>
-
-        <pre class="example highlight html">
-  &lt;link rel="preload" href="/assets/font.woff" as="font" loadpolicy="next"&gt;
-  &lt;link rel="preload" href="/assets/logo.webp" as="image" loadpolicy="next inert"&gt;
-        </pre>
-
-        <dl>
-          <dt><dfn>next</dfn></dt>
-          <dd>
-            Indicates that the specified resource may be required by the next navigation context (i.e. <dfn>speculative preload</dfn>), and that the user agent SHOULD fetch the resource if possible.
-
-            <p>Resource fetches that may be required for the next navigation can negatively impact the performance of the current navigation context due to additional contention for the CPU, GPU, memory, and network resources. To address this, the user agent SHOULD implement logic to reduce and eliminate such contention:</p>
-
-            <ul>
-              <li>Resource fetches required for the next navigation SHOULD have lower relative priority and SHOULD NOT block or interfere with resource fetches required by the current navigation context.</li>
-
-              <li>The optimal time to initiate a resource fetch required for the next navigation is dependent on the negotiated transport protocol, users current connectivity profile, available device resources, and other context specific variables. The user agent is left to determine the optimal time at which to initiate the fetch - e.g. the user agent MAY decide to wait until all other downloads are finished, or MAY choose to pipeline requests with low priority if the negotiated protocol supports the necessary primitives. Alternatively, the user agent MAY opt-out from initiating the fetch due to resource constraints, user preferences, or other factors.</li>
```

Comment:
Can we add the above paragraphs (or something similar) to the new "prefetch" section?